docs/cloud-native-security/cloud-workload-protection.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[cloud-workload-protection]]
-= Cloud workload protection
+= Cloud workload protection (VMs)
 
-Cloud workload protection helps you monitor and protect your Linux hosts and Kubernetes runtimes. It uses the <<install-endpoint,{elastic-defend}>> integration to capture cloud workload telemetry containing process, file, and network activity. In Kubernetes environments, this telemetry is further enriched with metadata that enables you to isolate events in your cloud topography.
+Cloud workload protection helps you monitor and protect your Linux VMs and Kubernetes runtimes. It uses the <<install-endpoint,{elastic-defend}>> integration to capture cloud workload telemetry containing process, file, and network activity. In Kubernetes environments, this telemetry is further enriched with metadata that enables you to isolate events in your cloud topography.
```

Comment:
Let's actually say just Linux VMs. We are deprecating support for Kubernetes clusters with the endpoint integration in favor of Defend for Containers.